docs/extras/modules/data_connection/vectorstores/integrations/azuresearch.ipynb

```diff
@@ -211,15 +211,372 @@
+   "source": [
+    "res = vector_store.similarity_search(query=\"Test 3 source1\", k=3, search_type=\"hybrid\", filters=\"source eq 'A'\")\n",
+    "res"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create a new index with a Scoring Profile"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.search.documents.indexes.models import (\n",
+    "    SearchableField,\n",
+    "    SearchField,\n",
+    "    SearchFieldDataType,\n",
+    "    SimpleField,\n",
+    "    ScoringProfile,\n",
+    "    TextWeights,\n",
+    "    ScoringFunction,\n",
+    "    FreshnessScoringFunction,\n",
+    "    FreshnessScoringParameters\n",
+    ")\n",
+    "\n",
+    "embeddings: OpenAIEmbeddings = OpenAIEmbeddings(model=model, chunk_size=1)\n",
+    "embedding_function = embeddings.embed_query\n",
+    "\n",
+    "fields = [\n",
+    "    SimpleField(\n",
+    "        name=\"id\",\n",
+    "        type=SearchFieldDataType.String,\n",
+    "        key=True,\n",
+    "        filterable=True,\n",
+    "    ),\n",
+    "    SearchableField(\n",
+    "        name=\"content\",\n",
+    "        type=SearchFieldDataType.String,\n",
+    "        searchable=True,\n",
+    "        retrievable=True,\n",
+    "    ),\n",
+    "    SearchField(\n",
+    "        name=\"content_vector\",\n",
+    "        type=SearchFieldDataType.Collection(SearchFieldDataType.Single),\n",
+    "        searchable=True,\n",
+    "        dimensions=len(embedding_function(\"Text\")),\n",
+    "        vector_search_configuration=\"default\",\n",
+    "    ),\n",
+    "    SearchableField(\n",
+    "        name=\"metadata\",\n",
+    "        type=SearchFieldDataType.String,\n",
+    "        searchable=True,\n",
+    "        retrievable=True,\n",
+    "    ),\n",
+    "    # Additional field to store the title\n",
+    "    SearchableField(\n",
+    "        name=\"title\",\n",
+    "        type=SearchFieldDataType.String,\n",
+    "        searchable=True,\n",
+    "        retrievable=True,\n",
+    "    ),\n",
+    "    # Additional field for filtering on document source\n",
+    "    SearchableField(\n",
+    "        name=\"source\",\n",
+    "        type=SearchFieldDataType.String,\n",
+    "        searchable=True,\n",
+    "        retrievable=True,\n",
+    "        filterable=True,\n",
+    "    ),\n",
+    "    # Additional data field for last doc update\n",
+    "    SimpleField(\n",
+    "        name=\"last_update\",\n",
+    "        type=SearchFieldDataType.DateTimeOffset,\n",
+    "        searchable=True,\n",
+    "        retrievable=True,\n",
+    "        filterable=True\n",
+    "    )\n",
+    "]\n",
+    "# Adding a custom scoring profile with a freshness function\n",
+    "sc_name = \"scoring_profile\"\n",
+    "sc = ScoringProfile(\n",
+    "    name=sc_name,\n",
+    "    text_weights=TextWeights(weights={\"source\": 5}),\n",
```

Comment:
Using weights to emphasize title over content would be a better example than using the "source" field, which might not need to be searhable at all.